wetterdienst/ui/cli.py

```diff
@@ -240,10 +243,10 @@ def cli():
         wetterdienst values --provider=dwd --network=observation --parameter=kl --resolution=daily --period=recent --station=1048,4411 --tidy
 
         # Limit output to specific date
-        wetterdienst values --provider=dwd --network=observation --parameter=kl --resolution=daily --period=recent --date=2020-05-01 --station=1048,4411
+        wetterdienst values --provider=dwd --network=observation --parameter=kl --resolution=daily --date=2020-05-01 --station=1048,4411
```

Comment:
Those changes to the help text will seriously conflict with my other patch GH-851, where I think I've fixed most of those. Maybe just remove those adjustments here?